### PR TITLE
Feature/39524

### PR DESCRIPTION
**Pull request by harshsuccessive**
created validation handler

### CODE REVIEW

File: src/controllers/trainee/validation.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+             required: true,
+             string: true,
+             in:['body'],
+             custom: function(value) {
+                 console.log('Value', value);
+                 throw { 
+                     error: 'Error Occured', message: 'Message' 
+                 }
+            }
+        },
```
Comment (namanpar) [inline src/controllers/trainee/validation.ts:13]: is this is validation file or config file

File: src/libs/validationHandler.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    const query = Object.keys(req.query).length;
+    keys.forEach((key) => {
+        const item = config[key];
+        const value = item.in.map((val) => {
+            return req[val][key];
+        });
+        console.log(value);
+        console.log(req.method);
+        if (item && item.required) {
+            if(req.method == 'POST' || req.method == 'PUT'){
```
Comment (namanpar) [inline src/libs/validationHandler.ts:28]: use es6 and proper naming convention

**Review by namanpar — CHANGES_REQUESTED** [review]

File: src/controllers/trainee/validation.ts (hunk 1 above)
Comment (harshsuccessive) [reply src/controllers/trainee/validation.ts:13]: it is a validation file and inside a constant config is defined as per the guidelines of user story on redmine

File: src/libs/validationHandler.ts (hunk 2 above)
Comment (harshsuccessive) [reply src/libs/validationHandler.ts]: used es6 and and named it please check

**Review by namanpar — APPROVED** [review]

**Pull request merged**